src/tools/sitemap/generate-sitemap.js

```diff
@@ -0,0 +1,115 @@
+/* eslint no-console: 0 */
+const fs = require('fs');
+const globby = require('globby');
+const prettier = require('prettier');
+const gemeenteData = require('../../data/gemeente_veiligheidsregio.json');
+
+// regioData being generated as we can't import an ES export into CommonJS
+const regioData = [...Array(25).keys()].map(
+  (n) => `VR${(n + 1).toString().padStart(2, '0')}`
+);
+
+/**
+ * Generates an xml sitemap depending on the given locale.
+ *
+ * @param locale
+ */
+const generateSitemap = async function (locale) {
+  console.log(`Generating sitemap '${locale || 'nl'}'`);
+  const prettierConfig = await prettier.resolveConfig('./.prettierrc.js');
+
+  const domain = `${
+    process.env.NEXT_PUBLIC_LOCALE === 'en' ? 'government' : 'rijksoverheid'
+  }`;
+
+  // Ignore Next.js specific files and API routes.
+  const pages = await globby([
+    './src/pages/**/*{.tsx,.mdx}',
+    '!./src/pages/_*.tsx',
+    '!./src/pages/api',
+  ]);
+
+  const paths = pages.map((page) =>
+    page
+      .replace('./src/pages', '')
+      .replace('.tsx', '')
+      .replace('/index.tsx', '')
+      .replace('/index', '')
+  );
+
+  const priorities = [
+    { path: '404', value: 0 },
+    { path: '500', value: 0 },
+    { path: 'landelijk', value: 0.8 },
+    { path: 'gemeente', value: 0.8 },
+    { path: 'regio', value: 0.8 },
+  ];
+
+  const pathsWithPriorities = paths.map((path) => {
+    const priority = priorities.find((priority) =>
+      path.includes(priority.path)
+    );
+
+    return {
+      path: path,
+      priority: priority != undefined ? priority.value : 0.6,
```

Comment:
nitpick; !==

Or maybe better
```suggestion
      priority: priority?.value ?? 0.6
```
But now I doubt if ?? is available outside of TS

Reply:
TS is not the issue, it's that NodeJS needs to support this, but that's easy to try out. These files and the webpack config file are not transpiled.

Reply:
Yes, that's what I mean. The current version of javascript (supported by Node.js) is likely not recognizing this. You'll have to stick with the node version used in the build pipeline I guess.

Reply:
> Yes, that's what I mean. The current version of javascript (supported by Node.js) is likely not recognizing this. You'll have to stick with the node version used in the build pipeline I guess.

If this is correct, the node version in use would be the one as defined in the Dockerfile from this repository. So there's a degree of control there.